cmd/client/main.go

```diff
@@ -51,7 +51,7 @@ var (
 	metricsAddr        = kingpin.Flag("metrics-addr", "Serve Prometheus metrics at this address").Default(":9369").String()
 	tokenPath          = kingpin.Flag("token-path", "Uses an OAuth 2.0 Bearer token found in this path to make scrape requests").String()
 	insecureSkipVerify = kingpin.Flag("insecure-skip-verify", "Disable SSL security checks for client").Default("false").Bool()
-	useLocalhost       = kingpin.Flag("use-localhost", "Use 127.0.0.1 to scrape metrics instead of FQDN").Default("false").Bool()
+	useLocalhost       = kingpin.Flag("use-localhost", "Use localhost to scrape metrics instead of FQDN").Default("false").Bool()
```

Comment:
I think I would prefer having a separate boolean flag for using ipv6, in general I think its better to have explict bindings